net/dns/resolver/forwarder_test.go

```diff
@@ -401,10 +399,8 @@ func runDNSServer(tb testing.TB, opts *testDNSServerOptions, response []byte, on
 }
 
 func enableDebug(tb testing.TB) {
+	tstest.Replace(tb, &verboseDNSForward, func() bool { return true })
 	const debugKnob = "TS_DEBUG_DNS_FORWARD_SEND"
```

Comment:
Can presumably remove the `const` here?

Reply:
yeah.

but then I also realized that even changing `&verboseDNSForward` can still race, so re-did it.